thoth/storages/graph/dgraph.py

```diff
@@ -763,6 +763,22 @@ def retrieve_unsolved_python_packages(self, solver_name: str) -> dict:
 
         return self._postprocess_retrieve_packages(result)
 
+    def retrieve_not_analyzed_python_package_version_entities(self) -> List[dict]:
+        """Retrieve a list of package names, versions and index urls that were not analyzed yet by package-analyzer."""
+        query = """{
+            f(func: has(%s), first: 10) @filter(NOT has(%s)) {
+                package_name:package_name
+                package_version:package_version
+                index_url:index_url
+            }
+        }""" % (
+            PythonPackageVersionEntity.get_label(),
+            PackageAnalyzerInput.get_name(),
+        )
+        result = self._query_raw(query)
+
+        return result["f"]
```

Comment:
I saw most of the methods call `return self._postprocess_retrieve_packages(result)`, but as in our case the output a little different because we also returning `index_url` as it find differ. That's way I didn't use `_postprocess_retrieve_packages` .

Reply:
Sounds good! :+1: